pkg/cgrouprate/cgrouprate.go

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package cgrouprate
+
+import (
+	"context"
+	"io"
+	"sync"
+
+	"github.com/cilium/tetragon/pkg/logger"
+	"github.com/cilium/tetragon/pkg/reader/notify"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	handle     *CgroupRate
+	handleLock sync.RWMutex
+)
+
+type Listener interface {
+	Notify(msg notify.Message) error
+	io.Closer
+}
```

Comment:
Is there a reason for defining a new type and not using oveserver.Listener https://github.com/cilium/tetragon/blob/25a0050cda943d8744b3f846d8c4a82219bce2da/pkg/observer/listener.go#L17?

Reply:
hm I think I did not want the observer dependency, I will check